Sources/Sentry/SentryHttpTransport.m

```diff
@@ -76,8 +80,16 @@ - (void)    sendEvent:(SentryEvent *)event
 // TODO: needs refactoring
 - (void)sendEnvelope:(SentryEnvelope *)envelope
    withCompletionHandler:(_Nullable SentryRequestFinished)completionHandler {
-    // TODO: Apply type based rate limiting
-    if (![self isReadyToSend:@""]) {
+    
+    if (![self.options.enabled boolValue]) {
```

Comment:
m: I'd really like to get rid of this flag. The SDK did a lot of work by the time we get this far. We should deal with this enabled/disable state at the API entrypoint and assume it's enabled after that.

Reply:
Yes I also don't like this. I didn't want to change it now. We should fix this soon.